### DISCUSSION TITLE
admin user how to activate new user

### DISCUSSION THREAD

**Discussion by TE-Caojianxun** [open-webui/open-webui#1571] [Q&A] [answered]
when I sign use another user, page pop-up:

  ```
Account Activation Pending Contact Admin for WebUI Access
Your account status is currently pending activation. To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel.
```

but I don't know how to activate new user  in admin user?

**Comment by peuportier** [comment, marked as answer]
Hey hey . Just log with your admin account then on the bottom left click on your user name , you will have a menu show up with admin panel .Click on it . Then you will have the list of users, on left you will see admin and the user pending . Click on pending to make them become user or admin or pending again. 
<img width="340" alt="Screenshot 2024-04-16 at 19 27 47" src="https://github.com/open-webui/open-webui/assets/30169867/87881261-0ea8-4c47-84e7-e1d05ca73833">
